code/modules/mob/living/simple_animal/animals/corgi.dm

```diff
@@ -38,7 +38,7 @@
 	real_name = "Ian"	//Intended to hold the name without altering it.
 	gender = MALE
 	desc = "It's a corgi."
-	tt_desc = "Canis commandus"
+	tt_desc = "Canis lupus familiaris commandus"
```

Comment:
the fact that i had to read the words 'Canis commandus' makes me want to never again even glance at this codebase